src/base/ERC20Standard.sol

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MIT 
+
+pragma solidity =0.7.6;
+
+import {ERC20} from '@openzeppelin/contracts/token/ERC20/ERC20.sol';
+
+contract ERC20Standard is ERC20 {
```

Comment:
should we add ability to modify decimals?

Reply:
Not necessary, 18 should be the default